### DISCUSSION TITLE
Plugins

### DISCUSSION THREAD

**Discussion by Asharalikhan4** [zed-industries/zed#6889] [Feature Requests]
How can i add plugins in zed? for example the most common we have live-server, how can i use that in zed.

**Comment by mikayla-maki** [comment]
We don’t have plugins yet.

**Reply by simonrouse9461** [reply]
Is there a plan to support plugins?

**Reply by SomeoneToIgnore** [reply]
The first step of this plan is https://github.com/zed-industries/zed/issues/7096 , future steps are less defined yet, but sure we do want to have language plugins.
